scripts/snap.py

```
@@ -0,0 +1,217 @@
+            out = OUT_DIR / f"snap-{slug}-{v}.png"
+            page.screenshot(path=str(out), full_page=args.full_page)
+            written.append(out)
+            print(f"  {v} → {out.relative_to(ROOT)}")
+
+        browser.close()
+
+    print(f"\nsnapped {len(written)} variants of {slug}")
+
+    if args.open_after and written:
+        if sys.platform == "darwin":
+            subprocess.run(["open", *map(str, written)], check=False)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)